client-app/src/css/AddProgramPage.css

```diff
@@ -1,39 +1,72 @@
 .container {
     display: flex;
-    flex-direction: column;
-    align-items: center;
-    margin-top: 20px;
+    justify-content: center;
 }
 
+/* Styling for the form as a grid layout */
 .form {
-    width: 400px;
+    display: grid;
+    grid-template-columns: 1fr; /* Single-column grid for all fields */
+    gap: 20px; /* Space between each grid item */
+    width: 50vw;
+    max-width: 500px;
+    background-color: #212430; /* Darker background for the form */
+    padding: 15px 25px 15px 25px;
+    border-radius: 25px;
+    box-shadow: 0 4px 8px rgba(63, 60, 60, 0.1); /* Adding box shadow for better visibility */
+    margin-top: 30px;
 }
 
-.form-group {
-    margin-bottom: 15px;
+/* Styling for the title inside the form */
+.heading {
+    color: #66fcf1; /* Sky blue color for the heading */
+    text-align: center;
+    font-size: 28px; /* Font size for the heading */
+    grid-column: 1 / -1; /* Span the entire width of the grid */
+    margin-bottom: 5px;
 }
 
 .label {
     display: block;
     margin-bottom: 5px;
+    color: #ffffff; /* Label text color to stand out against the dark background */
 }
 
+/* Red asterisk for required fields */
+.required {
+    color: red;
+}
+
+/* Styling for input and textarea fields */
 .input,
 .textarea {
     width: 100%;
-    padding: 8px;
-    border: 1px solid #ccc;
-    border-radius: 4px;
+    padding: 6px !important;
+    margin-top: 4px;
+    margin-bottom: 0 !important;
+    border-radius: 8px;
+    border: 1px solid #66fcf1;
+    font-size: 10px;
+    background-color: #f9f9f9;
+    color: #333;
 }
 
+/* Increasing size of description and aimAndCause text areas */
+.textarea {
+    min-height: 100px;
+}
+
+/* Centering the button group */
 .button-group {
     display: flex;
     justify-content: center;
+    gap: 10px; /* Add space between the buttons */
+    grid-column: 1 / -1; /* Span the entire width of the grid */
 }
 
+/* Styling for save and clear buttons */
 .save-button,
 .clear-button {
```

Comment:
I would style your clear button to change colors when it is hovered over, like you have with save. Hopefully that may quell any fears that a user may have regarding whether the button is working.